src/app/master-view/master-view.component.html

```diff
@@ -2,24 +2,29 @@
   <div class="column-layout group_1">
     <div class="column-layout group_2">
       <igx-list class="list">
-        <igx-list-item [isHeader]="false" *ngFor="let item of northwindJasonApiCustomers">
+        <igx-list-item
+          *ngFor="let item of northwindJasonApiCustomers"
+          [isHeader]="false"
+          [class.selected]="this.selectedCustomerId === item.customerID"
+          (click)="this.selectCustomer(item.customerID)"
+        >
           <span igxListLineTitle>{{ item.contactName }}</span>
           <span igxListLineSubTitle>{{ item.customerID }}</span>
         </igx-list-item>
       </igx-list>
     </div>
   </div>
-  <div class="column-layout group_3">
+  <div class="column-layout group_3" *ngIf="selectedCustomer">
```

Comment:
I believe this can be driven by the "data-context" approach.

Reply:
Yes, it's equivalent of a repeat of a single element.
So, 0-1 occurrences of it.